posthog-react-native/src/native-deps.tsx

```diff
@@ -40,12 +41,13 @@ export const getAppProperties = (): PostHogCustomAppProperties => {
   }
 
   if (OptionalReactNativeDeviceInfo) {
-    properties.$app_build = returnPropertyIfNotUnknown(OptionalReactNativeDeviceInfo.getBuildIdSync())
+    properties.$app_build = returnPropertyIfNotUnknown(OptionalReactNativeDeviceInfo.getBuildNumber())
```

Comment:
This matches the right app build when using expo-device and the Android/iOS SDKs
getBuildId != BuildNumber

Reply:
it was returning `UP1A.231105.003` instead of `1` which is the app build